de.bwravencl.ControllerBuddy.yml

```diff
@@ -0,0 +1,139 @@
+id: de.bwravencl.ControllerBuddy
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+command: launch.sh
+finish-args:
+  - --device=all
+  - --env=JAVA_TOOL_OPTIONS=-Dawt.useSystemAAFontSettings=gasp
+  - --filesystem=xdg-documents
+  - --persist=.java
+  - --share=network
+  - --socket=x11
+  - --share=ipc
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --talk-name=org.freedesktop.Flatpak
+modules:
+  - name: ControllerBuddy
+    buildsystem: simple
+    build-commands:
+      - |
+        cp -R ControllerBuddy/* /app/ &&
+        install -Dm 755 launch.sh -t /app/bin &&
+        install -Dm 644 icon.svg "/app/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg" &&
+        install -Dm 644 "$FLATPAK_ID.desktop" -t /app/share/applications &&
+        install -Dm 644 "$FLATPAK_ID.metainfo.xml" -t /app/share/metainfo &&
+        cp -R ControllerBuddy-Profiles /app/share/
+    sources:
+      - type: archive
+        only-arches:
+          - x86_64
+        url: https://github.com/bwRavencl/ControllerBuddy/releases/download/1.3.66-639d789/ControllerBuddy-linux-x86-64-1.3.66-639d789.tgz
+        sha256: f139aecc5f16575d32b614b0abed585a28b2e27c9b89d8918cc1ed044bbbda79
+        strip-components: 0
+        x-checker-data:
+          type: anitya
+          project-id: 376393
+          stable-only: false
+          url-template: https://github.com/bwRavencl/ControllerBuddy/releases/download/$version/ControllerBuddy-linux-x86-64-$version.tgz
+      - type: archive
+        only-arches:
+          - aarch64
+        url: https://github.com/bwRavencl/ControllerBuddy/releases/download/1.3.66-639d789/ControllerBuddy-linux-aarch64-1.3.66-639d789.tgz
+        sha256: b9896700fd131ce78295020bea5b6e2e71ce158099552b0155d1f21b1c6a8b30
+        strip-components: 0
+        x-checker-data:
+          type: anitya
+          project-id: 376393
+          stable-only: false
+          url-template: https://github.com/bwRavencl/ControllerBuddy/releases/download/$version/ControllerBuddy-linux-aarch64-$version.tgz
+      - type: file
+        url: https://raw.githubusercontent.com/bwRavencl/ControllerBuddy/refs/tags/1.3.66-639d789/icon.svg
+        sha256: 22cb081b000a30f248e6c5f842a9519d933339d01449fe2889e3379cc5628523
+        x-checker-data:
+          type: anitya
+          project-id: 376393
+          stable-only: false
+          url-template: https://raw.githubusercontent.com/bwRavencl/ControllerBuddy/refs/tags/$version/icon.svg
+      - type: script
+        dest-filename: launch.sh
```

Comment:
i think this will be easier to maintain as a separate file, long lines, yaml and no highlight makes it hard to read.